.github/workflows/ci-lite-tests.yml

```diff
@@ -72,20 +80,11 @@ jobs:
       run: |
         python .actions/assistant.py replace_oldest_ver
 
-    - name: switch PyTorch URL
-      run: python -c "print('TORCH_URL=https://download.pytorch.org/whl/' + str('test/cpu/torch_test.html' if '${{matrix.release}}' == 'pre' else 'cpu/torch_stable.html'))" >> $GITHUB_ENV
-
-    - name: Install package
-      env:
-        PACKAGE_NAME: ${{ matrix.pkg-name }}
+    - name: Adjust PyTorch versions in requirements files
```

Comment:
I believe this step was missing

Reply:
So, it seems it's a combination of these two issues:
This step was missing, but also changes made in #15337 were missing too

Reply:
Teamwork!